root/tree/index/runindex64.C

```diff
@@ -25,20 +27,22 @@ int runindex64(){
   tree->Branch("run", &run, "run/l");
   tree->Branch("event", &event, "event/l");
 
-  ULong64_t events[] = { 1,2,3, bigval, biguval, 5 };
-  run = 5;
+  ULong64_t   runs[] = { 8,5,5,5,      0,       0,  5,      4, 6, biguval,          1, maxMajor, maxMajor+1};
+  ULong64_t events[] = { 0,1,3,2, bigval, biguval,  5, bigval, 3,  bigval, maxMinor+1, maxMinor, 0};
```

Comment:
This is a weird layout, did this go through `clang-format`?

Reply:
That's actually a nice layout here where you can see the pair of values that goes together without having to count commas.